site-templates/pages/appRegistration.gomd

```diff
@@ -0,0 +1,22 @@
+<div class="row">
+
+  <div class="col-md-8 content">
+  
+    <h1>Application Registration</h1>
+
+    <h3>Overview</h3>
+    <p>
+    Application Registration enables a user to register for a Bluescape's account and to create an application in Bluescape. Upon creating an application,
+    the user is able to configure the application; such as, viewing the application details, regenerating client secrets, editing/deleting the application, etc.  
+    To register for an account, you will need to visit the Application Registration link indicated below. If you are interested in learning more about the Application Authorization process, please visit 
+    <a href="/docs/page/app-auth">OAuth flow</a> for more information.
+    </p>
```

Comment:
Should we place the general usefulness of the Application in here? The exact excerpt from the app auth page I'm referring to is this:

>  Applications that want to query the Bluescape API must be authorized to do so by an active Bluescape user. Applications obtain this authorization using the OAuth flow which enables users to share access to their account without divulging their password.

> Once authorized, applications receive access tokens from Bluescape which enable them to access Bluescape resources on behalf of the authorizing user.

Reply:
Since we already included a link to app auth, do we want to re-emphasize the above in app registration page?

Reply:
Maybe, if it's useful for explaining what Application Registration is used for...I think it's okay to re-emphasize it, if you think it's useful for delivering a quick explanation of what App Registration is...or perhaps a slight re-wording?